versions/3.0.md

```diff
@@ -989,7 +989,7 @@ When `example` or `examples` are provided in conjunction with the `schema` objec
 
 Field Name | Type | Description
 ---|:---:|---
-<a name="parameterContent"></a>content | Map[`string`, [Media Type Object](#mediaTypeObject)] | A map containing the representations for the parameter. The key is the media type and the value describes it.  The map MUST only contain one entry.
+<a name="parameterContent"></a>content | Map[`string`, [Media Type Object](#mediaTypeObject)] | A map containing the representations for the parameter. The key is the media type and the value describes it.  The key can be a single media type (e.g. `application/json`), a wildcard media type (e.g. `image/*`), or a comma-separated list of the two types. The map MUST only contain one entry.
```

Comment:
Hmm, my suggestion is "The key can be singular (e.g. `application/json`), a wildcard (e.g. `image/*`), or a comma-separated list of both (e.g. `image/*, application/json, text/plain`).".

I'm not sure it's much better though!